tests/integration/linux_bridge_test.py

```diff
@@ -375,6 +376,23 @@ def test_activate_empty_bridge_does_not_blocked_by_dhcp():
         assertlib.assert_state(desired_state)
 
 
+def test_port_vlan_not_implemented(port0_up):
+    bridge_name = TEST_BRIDGE0
+    port_name = port0_up[Interface.KEY][0][Interface.NAME]
+    port_vlan_config = {
+        LinuxBridge.Port.VLAN_SUBTREE: {
+            LinuxBridge.Port.Vlan.TYPE: LinuxBridge.Port.Vlan.ACCESS_TYPE,
+            LinuxBridge.Port.Vlan.TAG: 101,
+        }
+    }
+
+    bridge_state = _create_bridge_subtree_config((port_name,))
+    bridge_state[LinuxBridge.PORT_SUBTREE][0].append(port_vlan_config)
+    with pytest.raises(NmstateNotImplementedError):
```

Comment:
The only minor thing I can think of is adding a reference to track the actual implementation, so maybe a link to a Jira card.

Reply:
Hmm... you can move this to an xfail decorator and add there this info.